dpdispatcher/ssh_context.py

```diff
@@ -261,6 +261,12 @@ def bind_submission(self, submission):
         # self.remote_root = os.path.join(self.temp_remote_root, self.submission.submission_hash, self.submission.work_base )
         self.remote_root = pathlib.PurePath(os.path.join(self.temp_remote_root, self.submission.submission_hash)).as_posix()
 
+        try:
+            sftp = self.ssh_session.ssh.open_sftp()
+            sftp.mkdir(self.remote_root)
+        except OSError:
```

Comment:
Is `OSError` for line 265 or line 266?

Reply:
266 line. I will fix it.

Reply:
I don't remember - why do we need to create a directory here?